### DISCUSSION TITLE
Make email subjects configurable

### DISCUSSION THREAD

**Discussion by Lurkars** [pretix/pretix#2405] [Ideas] [answered]
In email content tab, only the message body can be set. I would like to change the subject of emails too (with placeholder support).

**Comment by raphaelm** [comment]
I've blocked this for a long time because I thought it would add too many settings, which is always annoying. But I'm getting less firm on it with the years.

**Comment by listerr** [comment, marked as answer]
Looks like this is now possible? 

https://pretix.eu/about/en/blog/20221130-release-4150/

> Customizable email subjects
> pretix has always allowed you to customize the content of all emails we send to your customers, but up until now, there has been a limitation: You could only change the text of the email. Starting with 4.15, we're bringing a much-requested improvement that allows you adjust the subject line of every email we send!
